src/isar/apis/schedule/pause_mission.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from http import HTTPStatus
+from typing import Optional
+
+from fastapi import Response
+from injector import inject
+
+from isar.config.settings import settings
+from isar.models.communication.queues.queue_timeout_error import QueueTimeoutError
+from isar.models.communication.queues.queues import Queues
+from isar.services.utilities.queue_utilities import QueueUtilities
+from isar.services.utilities.scheduling_utilities import SchedulingUtilities
+from isar.state_machine.states_enum import States
+
+
+class PauseMission:
```

Comment:
Could this be combined in a mission utilities file or class?

Reply:
Addressed in #274